bundles/AdminBundle/Resources/views/Admin/Login/lostpassword.html.twig

```diff
@@ -3,11 +3,19 @@
 {% block content %}
 
     {% if app.request.method == 'POST' %}
-        <div class="text error">
-            {{ 'A temporary login link has been sent to your email address.'|trans([],'admin') }}
-            <br/>
-            {{ 'Please check your mailbox.'|trans([],'admin') }}
-        </div>
+        {% if success == true %}
+            <div class="text success">
+                {{ 'A temporary login link has been sent to your email address.'|trans([],'admin') }}
+                <br/>
+                {{ 'Please check your mailbox.'|trans([],'admin') }}
+            </div>
+        {% else %}
+            <div class="text error">
+                {{ 'An error occured when resetting password:'|trans([],'admin') }}
```

Comment:
alternativaly, could leave the` error` as `?string` and replace with 

```
                {% if error == 'user_reset_password_too_many_attempts' %}
                    <b>{{ 'user_reset_password_too_many_attempts'|trans([],'admin') }}</b>
                {% else %}
                    <b>{{ 'There is no valid account with the info you provided'|trans([],'admin') }}</b>
                {% endif %}
```

Reply:
or just return response with `TooManyLoginAttemptsAuthenticationException` from the controller? like Authenticator https://github.com/pimcore/pimcore/blob/20ed632f4544fc5bd56cdceae50f17962d179e7f/bundles/AdminBundle/Security/Authenticator/AdminAbstractAuthenticator.php#L71

Reply:
In this particular case, it doesn't seem worth letting it throw the `TooManyLoginAttemptsAuthenticationException` nor `BruteforceProtectionException` (which is currently catched). 

In production it would display as
![image](https://user-images.githubusercontent.com/6014195/224353764-0daa1b5f-b6ff-4c4e-8405-32ee41593a63.png)
and is not very meaningful to the enduser.

Reply:
this works for me similar to Authenticator plain response:
```php
$limit = $limiter->consume(1);
if (false === $limit->isAccepted()) {
    $exception = new TooManyLoginAttemptsAuthenticationException(ceil(($limit->getRetryAfter()->getTimestamp() - time()) / 60));
    return new Response(strtr($exception->getMessageKey(), $exception->getMessageData()));
}
```

Reply:
yes, i recall that and made sense in there because it was on some event that redirected or so, and not in a controller that renders a template, but here we do have them and need them to differenciate a good/bad password reset outcome by passing a variable, should display this message more fancily than a text on white screen.

Reply:
I get it what you are trying to achieve but putting specific error logic on frontend template is not optimal. I would still try to keep it consistent no matter if the response comes from the authenticator or controller. Also Symfony suggests to return a blank response in the example [here](https://symfony.com/doc/current/rate_limiter.html#exposing-the-rate-limiter-status).

Reply:
The example is specific for REST API though, wouldn't consider it as best practice for this case.

In 11.x, we do output the error message fancily, so for consistency, the reset password throttling error  
- either should be the same as a blank response in login for 10.x but then apply the fancy message for 11 PR
- OR should convert the blank response  in 10.x (by storing it as a flash message in session or so) to match to what would look like in 11 and always display it fancily whenever a template is rendered. (imho this would be the best, at least from UI point of view)

![image](https://user-images.githubusercontent.com/6014195/224672657-a9494d9c-7454-424d-a8f1-e6d8f89c1b76.png)

See also
https://github.com/pimcore/pimcore/blob/f1d3ddb0b6e8941ffc423d913ce16c7fa95ca5fd/bundles/AdminBundle/templates/admin/login/login.html.twig#L7-L16

Reply:
> OR should convert the blank response in 10.x (by storing it as a flash message in session or so) to match to what would look like in 11 and always display it fancily whenever a template is rendered. (imho this would be the best, at least from UI point of view)

Let's do this then but first this PR needs to adapt 😉

Reply:
We should not show all messages together. 
![image](https://user-images.githubusercontent.com/5137917/224975316-348bd729-0c17-4c9e-9962-18c70b3bb0fd.png)

Reply:
like discussed earlier, please get rid of "no valid account" error message. please see https://github.com/pimcore/pimcore/pull/14568#discussion_r1132411779

Reply:
mhhh, weird why it appears twice? did you add some lines in the template for debugging?

AFAIU The problem is not about a "no vaid account" error message, but sharing the exact detail of why the account is not valid, eg. due missing password, due being inactive, due missing email etc... but keeping it on a generic would be fine.

Reply:
Not sure why it appeared twice. Running clear cache fixed the problem. 

About the error message, it should follow the statement `Return a consistent message for both existent and non-existent accounts.`.

Reply:
but in case, this line should be even more generic
https://github.com/pimcore/pimcore/pull/14568/files?w=1#r1135354877